### PR TITLE
Update pip installation guide

### PR DESCRIPTION
**Pull request by rettinghaus**

### CODE REVIEW

**Comment by rettinghaus** [comment]
@lpugin What about this one?

**Comment by lpugin** [comment]
Building from a local repository does not work? This is what I wanted to check but then I forgot.

**Comment by rettinghaus** [comment]
It was about generalizing for non-local repositories, but I see now, that we should also keep the statement with the local path.

File: _book/06-installing-or-building-from-sources/03-python.md [hunk 1]
```diff
@@ -111,14 +111,20 @@ python setup.py build_ext --inplace --define NO_PAE_SUPPORT
...
-You can build and install with `pip` directly from the repository with:
+You can build and install with `pip` directly from a remote repository with:
 
 ```bash
 pip install --global-option=build_ext git+https://github.com/rism-digital/verovio
 ```
 
 You may specify the branch, commit hash, or tag name after an `@` at the end of the Git url.
 
+If you have a local copy of the repository just run.
```
Comment (lpugin) [inline _book/06-installing-or-building-from-sources/03-python.md:122]:
```suggestion
If you have a local copy of the repository just run:
```

**Pull request merged**